src/components/textToSpeech/js/TextToSpeech.js

```diff
@@ -132,25 +130,55 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
             },
             handleError: "{that}.events.onError.fire",
             handlePause: {
-                changePath: "paused",
-                value: true
+                funcName: "fluid.textToSpeech.handlePause",
+                args: ["{that}"]
             },
             handleResume: {
-                changePath: "paused",
-                value: false
+                funcName: "fluid.textToSpeech.handleResume",
+                args: ["{that}"]
             }
         }
     });
 
+    // Issue commands to the speechSynthesis interface after a delay; this
+    // makes the wrapper behave somewhat better when issuing commands, especially
+    // play and pause
+    fluid.textToSpeech.asyncSpeechSynthesisControl = function (control, delay, args) {
+        setTimeout(function () {
+            speechSynthesis[control](args);
+        }, delay);
+    };
+
     fluid.textToSpeech.speak = function (that, speaking) {
         that.events[speaking ? "onStart" : "onStop"].fire();
     };
 
-    fluid.textToSpeech.pause = function (that, paused) {
-        if (paused) {
-            that.events.onPause.fire();
-        } else if (that.model.speaking) {
-            that.events.onResume.fire();
+    fluid.textToSpeech.handlePause = function (that) {
+        that.applier.change("paused", true);
+        that.events.onPause.fire();
```

Comment:
One (possible) argument to make for `onPause` and `onResume` events when they could be handled by model events is that it keeps a parallel non-model event structure in the component to the events in the wrapped WebSpeechSynthesis API.

Second possible point: it's a somewhat fine-grained (and possibly redundant) distinction but the model change represents a change to the component-tracked state of the speech, while the non-model events represent callbacks from the wrapped browser API about the state of the utterance.